index.js

```diff
@@ -13,10 +13,16 @@ const URL = require('url-parse');
 
 const dnsLookupP = promisify(dns.lookup);
 
-const checkHttp = async url => {
+const checkHttp = async (url, timeout) => {
 	let response;
 	try {
-		response = await got(url, {rejectUnauthorized: false});
+		response = await got(url, {
+			https: {
+				rejectUnauthorized: false
+			},
+			retry: 0,
```

Comment:
Is there a reason why we need to disable retries here? This is a pretty draconian way to check that a URL is reachable - many failures, such as ECONNRESET, can be intermittent.

I hit this while using another CLI tool that uses this library and refuses to do anything if one of the services it wants is "unreachable".